content/beginner/080_scaling/install_kube_ops_view.md

```diff
@@ -36,20 +43,25 @@ helm list
 should display : 
 ```
 NAME            REVISION        UPDATED                         STATUS          CHART                   APP VERSION     NAMESPACE
-kube-ops-view   1               Sun Sep 22 11:47:31 2019        DEPLOYED        kube-ops-view-1.1.0     0.11            default  
+kube-ops-view   1               Sun Sep 22 11:47:31 2019        DEPLOYED        kube-ops-view-1.1.4     20.4.0            default  
+```
+
+Next, run the kubectl port-forward to access the deployment in your local machine
+
+```
+kubectl port-forward $POD_NAME 8080:8080 > /dev/null &
```

Comment:
We should avoid using port-forward here and use the LB deploy with Kube-ops-view. If there is no LB by default we should pass parameters to helm to activate service type loadbalancer